src/components/pages/home/events.tsx

```diff
@@ -1,44 +1,124 @@
 'use client';
-import React, { useEffect, useRef, useState } from 'react';
+
+import { useState } from 'react';
 import events from '../../../data/events.json';
-import { MapPin } from '@phosphor-icons/react';
+import { MapPin } from 'phosphor-react';
```

Comment:
please update back to the newer version of @phophor-icons/react that we have added in as a dependency @VarshiniSuresh90